model/src/pyrenew/latent/infection_seeding_process.py

```diff
@@ -35,6 +42,11 @@ def __init__(
         self.infection_seed_method = infection_seed_method
         self.name = name
 
+        self.set_timeseries(
+            t_start=-self.infection_seed_method.n_timepoints,
```

Comment:
I don't think this should be hard-coded, though it is certainly a sensible default.